### DISCUSSION TITLE
Can I update the React examples?

### DISCUSSION THREAD

**Discussion by nattstack** [supabase/supabase#314] [Questions] [answered]
I would love to help out!
Can I update the `supabase` React examples?

I would update `react-scripts` to latest and remove some CRA boilerplate. I would also structure the React file differently like for example:

```JavaScript
export const TodoList
```

to 

```JavaScript
export default function TodoList() {
```

so imports would like

```JavaScript
import TodoList from './TodoList'
```

**Comment by kiwicopple** [comment, marked as answer]
go for it @nattwasm ! Thanks
